src/routes/safe/components/Apps/communicator.ts

```diff
@@ -64,6 +65,16 @@ class AppCommunicator {
     const hasHandler = this.canHandleMessage(msg)
 
     if (validMessage && hasHandler) {
+      trackSafeAppEvent({
+        event: GTM_EVENT.SAFE_APP,
+        name: this.app.name,
+        method: msg.data.method,
+        params: msg.data.params
+          ? JSON.stringify(msg.data.params).replaceAll(/0x[a-fA-F0-9]{40}/g, 'ethereum-address')
```

Comment:
Please add a comment `// Anonymize ethereum addresses`. It took a second to understand the code.

Reply:
I would extract the replacement text into a constant and use that instead so it is consistent. We use `SAFE_ADDRESS` in other events.

```suggestion
          ? JSON.stringify(msg.data.params).replaceAll(/0x[a-fA-F0-9]{40}/g, 'SAFE_ADDRESS')
```

Reply:
You will not be able to parse this in GA as well. They need to be separate keys.

Reply:
@iamacook you are making a replacement for SAFE_ADDRESS but at least in our case could be any ethereum address not only safe. Is not you case as well?

Reply:
If it is a Safe address, e.g. in the URL, I would use `SAFE_ADDRESS` otherwise `ETHEREUM_ADDRESS` perhaps? Just to be consistent.